### PR TITLE
Change copy on proposal creation

### PR DESCRIPTION
**Pull request by Ryun1**
- align character limit with CIP108
- Change copy to reflect CIP108 intent

### CODE REVIEW

File: src/components/CreationGoveranceAction/Step2.jsx [hunk 1]
```diff
@@ -475,7 +475,7 @@ const Step2 = ({
                                 label='Amount'
                                 type='tel'
                                 variant='outlined'
-                                placeholder='e.g. 2000'
+                                placeholder='e.g. 2000 ada'
```
Comment (Ryun1) [inline src/components/CreationGoveranceAction/Step2.jsx:478]: **Question**

Should this be in Ada or Lovelace? (I think Ada is fine for the frontend)
Which unit is the Wallet API expecting when creating the treasury withdrawal action?

@MSzalowski @vpaunovic

**Review by vpaunovic — APPROVED** [review]

File: src/components/CreationGoveranceAction/Step2.jsx (hunk 1 above)
Comment (vpaunovic) [reply src/components/CreationGoveranceAction/Step2.jsx:479]: @Ryun1 I approved the PR and pushed a commit with those max lengths for abstract, rationale, and motivation fields. Do you think we should wait for @MSzalowski's reply?

**Comment by vpaunovic** [comment]
@Ryun1 Can we merge this now, because the changes are already made on the backend side and they are connected with those changes on FE side so we can publish new PDF UI version ?

File: src/components/CreationGoveranceAction/Step2.jsx (hunk 1 above)
Comment (MSzalowski) [reply src/components/CreationGoveranceAction/Step2.jsx:479]: It should be ADA. I don't know if there are any amounts presented in Lovelace expected by wallet API

Comment (MSzalowski) [reply src/components/CreationGoveranceAction/Step2.jsx:479]: But not 100% sure to be honest :sweat_smile:

Comment (Ryun1) [reply src/components/CreationGoveranceAction/Step2.jsx:479]: lets do ada ! and hope make a note to recheck later

**Review by vpaunovic — APPROVED** [review]

**Pull request merged**